runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -317,6 +322,12 @@ public void onGameStateChanged(final GameStateChanged event)
 		{
 			chestLooted = false;
 		}
+		else if (event.getGameState() == GameState.LOGIN_SCREEN)
+		{
+			chestLooted = false;
+			panel.resetLootPanel();
```

Comment:
This will delete the loot upon logout, even the loot that previously has been saved to the server.